```
  C:\projects\cocos2d-x\cocos\renderer\CCRenderState.cpp(485): error C2039: 'toupper': is not a member of 'std' [C:\projects\cocos2d-x\win32-build\engine\cocos\core\cocos2d.vcxproj]
```

Miss including header files on windows.

--- a/cocos/renderer/CCRenderState.cpp
+++ b/cocos/renderer/CCRenderState.cpp
@@ -27,6 +27,7 @@
 
 #include "renderer/CCRenderState.h"
 
+#include <cctype>
 #include <string>
 
 #include "renderer/CCTexture2D.h"

